docs/contributing/pull-requests/branching.md

```diff
@@ -30,144 +31,111 @@ In this example, the full branch name would be `vault/pm-1234/update-csp-hashes`
+branch name or include the story key for all branches, depending on whether you are using
+subtask-level work breakdown.
+
+:::
+
 ### Branches for multiple Jira issues
 
 If the branch will contain work from multiple Jira issues (most likely due to it being a
-[Long-Lived Feature Branch](#long-lived-feature-branch)), the name should be a descriptive name of
+[long-lived feature branch](#long-lived-feature-branch)), the name should be a descriptive name of
 the feature, separated by dashes (e.g. `my-long-lived-feature`). Consider brevity when possible, as
-our QA team will need to use this branch name when performing QA testing on the feature.
+our QA team may need to use this branch name when performing QA testing on the feature.
 
 ## Branching for Development
 
-The branching strategy for development depends upon whether the work will be managed using what we
-have termed a "Long-Lived Feature Branch" or not.
-
-:::note
-
-We use the term "feature" in this documentation to refer to any change to the codebase, whether it
-be for a new set of functionality, a bug fix, or refactoring existing code. It is not meant to
-exclude types of development that are not "features" per se.
-
-:::
-
 ### Which Branching Model to Choose?
 
-The choice of development branching model depends on how you plan to handle the **entire testable,
-releasable feature**. As such, it should be planned in advance when beginning work on a new set of
-Jira stories or tasks, and it requires coordination across the entire team - Development, QA, and
-Product.
+The main point of consideration in choosing a branching model is whether the changes can be
+introduced directly into `master` with every pull request or whether they need to be merged into a
+long-lived feature branch for QA. As such, it should be planned in advance when beginning work on a
+new set of Jira stories or tasks, and it requires coordination across the entire team - Development,
+QA, and Product.
 
-Choose a **Long-Lived Feature Branch** if the feature will:
+Choose **branching and merging into `master`** if the feature will:
 
-- Require multiple Pull Requests to produce a testable, releasable change, and
-- It is not possible to put the changes behind a feature flag
+- Require a single pull request to produce a testable, releasable change, with QA taking place on
+  the PR branch, or
+- Allow multiple incremental pull requests **with the changes encapsulated behind a feature flag**,
+  with QA taking place in `master`.
 
-Choose a **Short-Lived Feature Branch** if the feature will:
+Choose **branching and merging into a long-lived feature branch** if the feature will:
 
-- Require a single Pull Request to produce a testable, releasable change, or
-- It is possible to put the changes behind a feature flag while multiple PRs are in flight.
+- Require multiple pull requests to produce a testable, releasable change, **and**
+- It is not possible to put the changes behind a feature flag
```

Comment:
(non-blocking) Are we practicing long-lived feature branches at all any more? Is there any continued justification for them? I prefer to just have one way to do things (that is, the correct way) when possible.

Reply:
I feel like there is still a scenario in which large features would like to be introduced together due to either technical or product-driven considerations, but we can't put them behind a feature flag.  The Vertical Vault Nav changes are an example.  Perhaps I can supplement this with a follow-up PR that extracts this into a callout that highlights how it's not our current best practice?